qa/common.check

```diff
@@ -1616,16 +1616,16 @@ _check_series()
 #
 _check_key_server_version()
 {
-    if which valkey-cli >/dev/null 2>&1
+    if which valkey-cli >/dev/null 2>&1 && pidof valkey-server >/dev/null 2>&1
```

Comment:
@kurik this is the first use of 'pidof' throughout the test suite ... are we a/ sure it exists on all platforms and b/ is always installed?

If not, there is a common function _get_pids_by_name that is used in quite a few places that may make a handy substitute.

Reply:
ad a/ no, I am not sure  :-)
ad b/ at least on Linux and BSD systems it should be part of the core system, but I do not have a way how to verify it

I am going to rework the patch using the proposed function .... thanks for the comment.